Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md

```diff
@@ -106,14 +106,14 @@ Following the same steps, let's add a few more properties to the *Content* group
 :::tip
 **Media Picker**
 
-This editor let's you upload or select an existing media item from the Media section, and add it to your content.
+This editor lets you upload or select an existing media item from the Media section and add it to your content.
 :::
 
 With these properties added our Document Type now looks something like this:
 
 ![More properties added to Document Type](images/DocType-More-Properties.png)
 
-Remember to save the Document Type by selecting the green **Save** button in the bottom-right.
+Remember to save the Document Type by clicking the green **Save** button in the bottom-right.
```

Comment:
We prefer using "selecting" over "clicking", as "selecting" covers both "clicking" and "tabbing" your way to the button.

```suggestion
Remember to save the Document Type by selecting the green **Save** button in the bottom-right corner.
```